dockstore-webservice/src/main/java/io/dockstore/webservice/resources/MetadataResource.java

```diff
@@ -79,6 +81,8 @@
 public class MetadataResource {
 
     private static final Logger LOG = LoggerFactory.getLogger(MetadataResource.class);
+    private static final int ERROR_STATUS_CODE = 500;
```

Comment:
This is a standard library value 
https://docs.oracle.com/javaee/7/api/javax/ws/rs/core/Response.Status.html#INTERNAL_SERVER_ERROR

Reply:
Still need this